akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala

```diff
@@ -349,7 +349,8 @@ class NewConnectionPoolSpec extends AkkaSpecWithMaterializer("""
 
       responseOutSub.request(1)
       acceptIncomingConnection()
-      val (Success(_), 42) = responseOut.expectNext()
+      val (Success(_), result) = responseOut.expectNext()
+      result shouldEqual 42
```

Comment:
Sad that this is no longer allowed :). I'd be fine with silencing those warnings for the test scope...

Reply:
Actually, I don't think those usages are forbidden, I just don't understand the warning or the solution :)

```
[warn] /home/johannes/git/opensource/akka-http/akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala:352:11: Pattern definition introduces Unit-valued member of $anon; consider wrapping it in `locally { ... }`.
```

Reply:
Ah, I know understood. Because with is wrapped in an anonymous `new TestSetup` this pattern match would actually create a field (which doesn't matter in the test).

Reply:
Extra braces around the statement helped.